lib/golomb/golomb.js

```diff
@@ -42,7 +41,8 @@ class Golomb {
   constructor() {
     this.n = 0;
     this.p = 0;
-    this.m = M;
+    this.M = new U64(784931);
+    this.m = this.M;
```

Comment:
I wonder if we can leave these `null` since this will be an abstract constructor?